EnglishFilter/sections/specific.txt

```diff
@@ -510,8 +510,8 @@ news18.com##div[style^="width:300px;min-height:250px;"]
 news18.com##div[style^="width: 300px; min-height: 250px;"]
 news18.com##div[style="min-height:527px"]
 news18.com##div[style="min-height: 527px;"]
-illink.net,owllink.net,birdurls.com###link-view > br
-illink.net,owllink.net,birdurls.com##.box-main > br
+shrinke.me,illink.net,owllink.net,birdurls.com###link-view > br
+shrinke.me,illink.net,owllink.net,birdurls.com##.box-main > br
```

Comment:
looks like these rules do nothing

Reply:
What! It works on my end. What do you suggest?

Reply:
Yes, you are right, now I see